packaging/nnstreamer.spec

```diff
@@ -76,6 +76,7 @@ BuildRequires:	pkgconfig(capi-privacy-privilege-manager)
 BuildRequires:	pkgconfig(capi-system-info)
 BuildRequires:	pkgconfig(capi-base-common)
 BuildRequires:	pkgconfig(dlog)
+BuildRequires:	pkgconfig(libmvnc)
```

Comment:
Although this is compatible with devel:Tizen, it's not compatible with Tizen;Unified. We need to merge mvnc libraries into Tizen;Unified first. NNStreamer's master branch should be compatible with Tizen:Unified.

Reply:
Right, We need to establish the Movidius Neural Computing (e.g., libmvnc)  git repository in Tizen:Unified  without devel:Tizen.

Reply:
How do you want the repo formed?

/adaptation/npu/mvnc?
/adaptation/npu/libmvnc?
/adaptation/npu/movidius?
???

Reply:
@myungjoo How about /adaptation/npu/libmvnc?

Reply:
and.. looking at platform/adaptation/* (probably they've changed the rule from adaptation/* --> platform/adaptation/* recently), we may need an additional naming rule:

platform/adaptation/npu/intel/libmvnc
platform/adaptation/npu/intel/openvino
platform/adaptation/npu/google/edgeTPU
...

or

platform/adaptation/npu/intel_libmvnc
platform/adaptation/npu/intel_openvino
platform/adaptation/npu/google_edgeTPU
...

Reply:
IMO, intel_* or google* looks pretty as well as samsung_exynos. About the name, npu, does it look a little bit specific? How about using the ambitious word, ml? 

```
platform/adaptation/ml/intel_libmvnc
```

Reply:
ml smells too much of software without hardware. In other words, "npu" makes people think of a hardware piece (thus some drivers or HALs) right away while "ml" makes people think of a pure software component (e.g., tensorflow, caffe).

Reply:
Let's try platform/adaptation/npu/intel_libmvnc

Reply:
> Let's try platform/adaptation/npu/intel_libmvnc

Got it. I will go with that name.

Reply:
https://bugs.tizen.org/browse/SO-1845

Reply:
> https://bugs.tizen.org/browse/SO-1845

You've already done it. Thanks.